mlos_bench/mlos_bench/service/remote/azure/azure_services.py

```diff
@@ -134,7 +136,7 @@ def __init__(self, config: dict, parent: Service):
             self.vm_start,
             self.vm_stop,
             self.vm_deprovision,
-            self.vm_reboot,
+            self.vm_restart,
```

Comment:
Part of planned rework for splitting VM deploy ops and Host/OS ops into separate types.